content/rc/rc-quickstart.md

```diff
@@ -114,7 +79,7 @@ These are displayed in the **Configuration** tab.
 
 Once you have the connection details, you can connect in a variety of ways, including:
```

Comment:
I would put a heading here to point out that we are now connecting to our database.